app/navigation/Navigator.scala

```diff
@@ -175,6 +181,12 @@ class Navigator @Inject()() {
       case false => routes.OrganisationAddressController.onPageLoad(mode)
     }
 
+  private def emailAddressQuestionRoutes(mode: Mode)(ua: UserAnswers): Option[Call] =
```

Comment:
There is a todo on line 175 to link to the email page, can you link those pages as part of this ticket?